docs/source/soundata.rst

```diff
@@ -27,6 +27,13 @@ TAU Urban Acoustic Scenes 2019
    :members:
    :inherited-members:
    
+TAU Urban Acoustic Scenes 2020 Mobile
```

Comment:
Can you add the dataset to the dataset list in the README?

Reply:
Yes!